safe-global/web-core
# Fix: avoid multiple pending txs with the same nonce

## What it solves

Resolves #1667

## How this PR fixes it

Disallows immediately executing a newly created tx w/o signatures if there are already other pending transactions.

## How to test
In a 1/X Safe, create and immediately execute a tx. Remember its nonce.
Quickly create another tx. Notice that it will have the same nonce as the previous tx, but the Execute option will be disabled.
